projects/aws/bottlerocket-bootstrap/pkg/service/secretsmanager.go

```diff
@@ -0,0 +1,49 @@
+package service
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+)
+
+var secretsManagerService SecretsManagerService
```

Comment:
Instead of a global service like this, can we re-architect this to follow golang recommended way using new(). Something like this
```
type SecretsManager struct {
    BaseClient *secretsmanager.Client
}

func NewSecretsManager() {
    secretsManager = new(SecretsManager)
    secretsManager.blah = foo
    return secretsManager...
}

func (sm *SecretsManager) DeleteSecret(...) {}
```
You can keep the interface.

Reply:
Yes, will do